examples/phoenix_app/lib/phoenix_app_web/api_spec.ex

```diff
@@ -10,7 +10,27 @@ defmodule PhoenixAppWeb.ApiSpec do
         version: "1.0"
       },
       servers: [OpenApiSpex.Server.from_endpoint(PhoenixAppWeb.Endpoint)],
-      paths: Paths.from_router(PhoenixAppWeb.Router)
+      paths: Paths.from_router(PhoenixAppWeb.Router),
+      components: %Components{
+        securitySchemes: %{
+          "oauth" => %SecurityScheme{
+            type: "oauth2",
+            description: "Authenticate with Github OAuth 2",
+            flows: %OAuthFlows{
```

Comment:
Shouldn't `%OAuthFlows{}` be `%{flow_type() => OAuthFlowObject.t()}`, where `flow_type` is `:implicit | :password | :client_credentials | :authorization_code``?

---

### OAuth Flows Object

Allows configuration of the supported OAuth Flows.

#### Fixed Fields

| Field Name | Type | Description |
| -- | -- | -- |
| implicit | OAuth Flow Object | Configuration for the OAuth Implicit flow |
| password | OAuth Flow Object | Configuration for the OAuth Resource Owner Password flow |
| clientCredentials | OAuth Flow Object | Configuration for the OAuth Client Credentials flow. Previously called application in OpenAPI 2.0. |
| authorizationCode | OAuth Flow Object | Configuration for the OAuth Authorization Code flow. Previously called accessCode in OpenAPI 2.0. |

https://swagger.io/specification/#securitySchemeObject

Reply:
Never mind, my mistake.